dbus-codegen/src/generate.rs

```diff
@@ -683,7 +683,9 @@ fn write_intf_crossroads(s: &mut String, i: &Intf, opts: &GenOpts) -> Result<(),
 fn write_module_header(s: &mut String, opts: &GenOpts) {
     *s += &format!("// This code was autogenerated with `dbus-codegen-rust {}`, see https://github.com/diwic/dbus-rs\n", opts.command_line);
     *s += &format!("use {} as dbus;\n", opts.dbuscrate);
-    *s += &format!("use {}::arg;\n", opts.dbuscrate);
+    if opts.serveraccess != ServerAccess::MethodInfo {
+        *s += &format!("use {}::arg;\n", opts.dbuscrate);
+    }
     if opts.futures {
```

Comment:
This one is incorrect. The `arg` module is used in many places, such as `arg::Variant` or `arg::Get`.

Reply:
I wasn't sure about this one myself! I'm gonna remove it.